filer/fields/file.py

```diff
@@ -41,8 +42,12 @@ def render(self, name, value, attrs=None):
         if not related_url:
             related_url = reverse('admin:filer-directory_listing-last')
         params = self.url_parameters()
+        params['_pick'] = 'file'
         if params:
-            lookup_url = '?' + '&amp;'.join(['%s=%s' % (k, v) for k, v in list(params.items())])
+            lookup_url = '?' + '&amp;'.join([
```

Comment:
why not use urlencode? you would need to make sure to encode this in utf8 in both scenarios